osmosis/assetlist.json

```diff
@@ -17974,6 +17974,25 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/COCA.png"
         }
       ]
+    },
+    {
+      "description": "Memecoin for The International Brane Wave",
```

Comment:
Why re-registering BRNZ?

Reply:
wasn't trying to, must've been an old commit